Gemfile

```diff
@@ -51,10 +51,14 @@ group :development, :test do
   gem 'factory_bot_rails'
   gem 'faker'
   gem 'rspec-rails'
-  gem 'webmock'
-  gem 'vcr'
   gem 'shoulda-matchers'
   gem 'travis'
+  gem 'webmock'
```

Comment:
When going through the webmock lesson it specifically said to not add webmock to development, so idk if this breaks anything

Reply:
Good catch, it has been removed and all tests are still working